documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -0,0 +1,95 @@
+# Performance Tests Methodology
+
+## Introduction
+This page describes methodology of Knotx application performance tests.
```

Comment:
How about

_This page describes *the* methodology of Knotx performance tests._